config/routes.rb

```diff
@@ -3,4 +3,6 @@
     root "static_pages#home"
     get "static_pages/help" 
     get "static_pages/contact" 
+    get "/signup", to: "users#new"
+    resources :users, only: %i(new create show)
```

Comment:
Cái new số 6 đã dùng router url khác rồi thì k cần ở resources rồi nhé
resources :users mặc định sẽ sinh ra 7 cái router tương ứng.
https://viblo.asia/p/resource-routing-trong-rails-RQqKLkbM57z

Nếu Hà xài của cái new ở đâu thì nó sẽ sinh ra thêm cái router là users/new cũng trỏ đến method users#new
Cái này k cần thiết vì đã xài /signup ở line 6 rồi. Nên bỏ đi
```suggestion
    resources :users, only: %i(create show)
```
Tham khảo thêm cái option except là ngược của cái only: